data/lang/quitconfirmation-core/en.json

```diff
@@ -0,0 +1,82 @@
+{
+   "QUIT" : {
+      "description" : "Quit game, as a question",
+      "message" : "Quit?"
+   },
+   "YES" : {
+      "description" : "Yes, quit program",
+      "message" : "Yes"
+   },
+   "NO" : {
+      "description" : "No, don't quit program",
+      "message" : "No"
+   },
+   "MSG_1" : {
+      "description" : "",
+      "message" : "Get out of here and go back to your boring programs?"
+   },
+   "MSG_2" : {
+      "description" : "",
+      "message" : "Don't go now, there's a Pirate waiting for you in the other program."
+   },
+   "MSG_3" : {
+      "description" : "",
+      "message" : "Are you sure you want to quit this great game that cost you nothing?"
+   },
+   "MSG_4" : {
+      "description" : "",
+      "message" : "Doest thou wish to leave with such hasty abandon?"
+   },
+   "MSG_5" : {
+      "description" : "(Inspired by Wolfenstein 3D)",
+      "message" : "Choose {no} for guns and glory, {yes} for work and worry."
+   },
+   "MSG_6" : {
+      "description" : "",
+      "message" : "Click {no} to be brave, click {yes} to cower in shame."
+   },
+   "MSG_7" : {
+      "description" : "Doom 1 anyone? Although it said DOS instead of Command line",
+      "message" : "I wouldn't leave if I were you. The command line is much worse."
+   },
+   "MSG_8" : {
+      "description" : "",
+      "message" : "I'm thinking you want to click {no} to play more. You do it."
+   },
+   "MSG_9" : {
+      "description" : "",
+      "message" : "Clicking {yes} means you must return to the humdrum workday world."
+   },
+   "MSG_10" : {
+      "description" : "",
+      "message" : "Please choose {no} to waste a few more productive hours. Please?"
+   },
+   "MSG_11" : {
+      "description" : "",
+      "message" : "Explorers don't quit, but go ahead and click {yes} if you aren't one."
+   },
+   "MSG_12" : {
+      "description" : "",
+      "message" : "Choose {no} If you are a though bounty hunter, if not, click {yes} daintily."
```

Comment:
Doh! should be "Tough". 
Thanks. 
Will fix.
(Hacked it all up today in "the between time" at work.)